webpack.config.js

```diff
@@ -23,6 +23,10 @@ module.exports = {
       {
         test: /\.md$/,
         loader: "html!markdown"
+      },
+      {
+        test: /\.css$/,
+        use: "css-loader"
```

Comment:
I guess you have missed adding the package.json

Reply:
Nope, the css-loader is already on the package.json